src/Skia/Avalonia.Skia/ImmutableBitmap.cs

```diff
@@ -88,6 +97,25 @@ public void Save(Stream stream)
         /// <inheritdoc />
         public void Draw(DrawingContextImpl context, SKRect sourceRect, SKRect destRect, SKPaint paint)
```

Comment:
This method might be called for RTB on the UI thread which will most likely cause a segfault

Reply:
What was the reason for not using a GPU backed texture for RTB?

Reply:
RTB drawing context is used from the UI thread and can not touch anything used by the renderer. If/when we enable gpu acceleration for rtb, it will also use a different GrContext.

Reply:
1) the code doesn't check if it's called from the renderer thread and that it can touch the GrContext
2) disabling GPU acceleration for RTBs is a temporary solution to avoid segfaults with the current implementation